soundfonts/LICENSE.txt

```diff
@@ -0,0 +1,25 @@
+*** GeneralUser GS v1.51 ***
+***      License v2.0     ***
+
+** License of the complete work **
+You may use GeneralUser GS without restriction for your own music creation, private or commercial.  This SoundFont bank is provided to the community free of charge.  Please feel free to use it in your software projects, and to modify the SoundFont bank or its packaging to suit your needs.
+
+** License of contained samples **
+GeneralUser GS inherits the usage rights of the samples contained within, all of which allow full use in music production, including the ability to make profit from musical recordings created with GeneralUser GS.
+
+Many of the samples are original, but some were taken from other banks freely (and legally) available on the Internet from various SoundFont websites.  Because GeneralUser GS originated as a personal project with no intention for publication, I cannot be 100% sure where all of the samples originated, although I do know that none of them came from commercially published SoundFont packages or sample CDs.  Regardless, many "free" SoundFonts available on the web may indeed contain samples of questionable origin.  My understanding of the copyrights of all samples is only as good as the information provided by the original sources. If you become aware of any restricted samples being used in GeneralUser GS, please let me know so I can replace them.
+
+This uncertainty may concern you if you intend to use GeneralUser GS in a commercial software product.  That being said, I have never received any complaint regarding sample ownership since I published the original GeneralUser GS back in 2000, and as far as I am aware, neither have any of the companies creating commercial software products using GeneralUser GS.
```

Comment:
This is really very vague and in fact not better than any other license for any other replacement soundfont suggested so far. Sorry, but I just can't stand "I have taken some samples from the Internet, but nobody has complained so far" type of disclaimers. Not sure what to think about this...

Reply:
All we have for TimGM6mb.sf2 is Tim Brechbill word: https://musescore.org/en/node/1463#comment-9168
I don't think a completely legal GM soundfont exist. If only Roland would release their sound font for public use...

Reply:
Yes, but at least that *is* a word. And the choice for this being distributable under GPL is backed up by varios Linux distros packaging it as their default soundfont.